com.unity.render-pipelines.universal/ShaderLibrary/UnityGBuffer.hlsl

```diff
@@ -87,9 +128,11 @@ FragmentOutput SurfaceDataToGbuffer(SurfaceData surfaceData, InputData inputData
     output.GBuffer0 = half4(surfaceData.albedo.rgb, PackMaterialFlags(materialFlags));   // albedo          albedo          albedo          materialFlags   (sRGB rendertarget)
     output.GBuffer1 = half4(surfaceData.specular.rgb, 0);                                // specular        specular        specular        [unused]        (sRGB rendertarget)
     output.GBuffer2 = half4(packedNormalWS, packedSmoothness);                           // encoded-normal  encoded-normal  encoded-normal  packed-smoothness
-    output.GBuffer3 = half4(globalIllumination, 0);                                      // GI              GI              GI              [not_available] (lighting buffer)
-    #if USE_SHADOWMASK
+    output.GBuffer3 = half4(globalIllumination, 1);                                      // GI              GI              GI              [optional: see OutputAlpha()] (lighting buffer)
+    #if OUTPUT_SHADOWMASK == 1
     output.GBuffer4 = unity_ProbesOcclusion;
```

Comment:
There is no need for this logic. inputData.shadowMask will have unity_ProbesOcclusion value if subtractive lighting is used. Basically inputData.shadowMask has baked shadows now

Reply:
Done.